content/en/seachat/seachat-manual/02-create-agent/04-agent-information.md

```diff
@@ -0,0 +1,114 @@
+
+### Interact with AI Agent
+You can immediately interact with the AI agent that you have just built. In **Agent URL**, you can click the URL to interact with your AI agent in a standalone window. 
+
+Or if you wish to integrate your AI agent into your website, you can also access your API key in the **API Key** section. Please check out the [Webpage Integration](/seachat/seachat-manual/04-channels/08-install-to-webpage/) for more information on how to programmatically integrate your AI agent into your website.
+
+<div style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+    <a href="/images/seachat/en/agent-information/additional-options.png" target="_blank">
+    <img width="80%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/en/agent-information/additional-options.png" alt="image that displays the additional options in Basic Settings">
+    </a>
+</div>
+    <p style="margin-top: 20px; font-size: 15px">Additional Options in Basic Settings</p>
+</div>
+
+## Advanced Settings
+
+Different from the basic settings, advanced settings allow you to customize your AI agent with more advanced features that SeaChat has curated for you. Inside **Advanced Settings** under **Agent Information**, the following advanced features are available:
+
+<div style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+    <a href="/images/seachat/en/agent-information/advanced-settings-dashboard.png" target="_blank">
+    <img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/en/agent-information/advanced-settings-dashboard.png" alt="image that displays the additional options in Basic Settings">
+    </a>
+</div>
+    <p style="margin-top: 20px; font-size: 15px">Advanced Settings Dashboard</p>
+</div>
+
+### Agent Memory
+Agent memory fields are used to extract information from a conversation and store it for future reference. Memory is a powerful yet complex feature. We recommend you refer to the [Agent Memory](/seachat/seachat-manual/02-create-agent/03-agent-memory/) tutorial to learn more about how to set up memory fields for your AI agent.
+
+### Retrieval Augmented Generation (RAG)
+Through Retrieval Augmented Generation (RAG), we can optimize the output of a Large Language Model (LLM) by integrating external data that is not present in the training data (Knowledge). This feature will drastically reduce the continuous training that you would otherwise need without RAG. Retrieval augmented generation is a sophisticated framework for improving the response of LLM models, and thus we invite you to consult our dedicated section on RAG for a better understanding of the feature.
```

Comment:
I think this is too technical. Let's just focus on why RAG settings provide ways for SeaChat users to customize **how knowledge is considered in real time retrieval**. You can explain why we'd want to customize RAG retrieval setting. It is because different RAG strategy works for different types of knowledge base. We want to allow user to experiment and customize the way they interact with knowledge base as much as possible. 


Retrieval Augmented Generation (RAG), we can optimize the output of SeaChat AI agent by integrating external data that is not present in the original training data (Knowledge). This feature will drastically reduce the continuous training that you would otherwise need without RAG.